blog/20230210-bbs-swe-course-basel/index.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "Workshop on good software engineering practice for R"
+subtitle: "Teaching foundations of SWE for statisticians with R"
+author: "SWE WG"
+description: Under the umbrella of the Basel Biometric Society, Daniel, Friedrich, and Kevin organized a one-day workshop on good SWE practice in R.
+date: "2023-02-13"
+categories: [news]
+image: "topics_and_people.jpg"
+---
+
+On Friday February 10, Daniel, Friedrich, and Kevin held
+a one-day workshop on good software engineering practice for R
+(GSWEP4R) in Basel.
+
+![](topics_and_people.jpg){width=90% fig-align="center"}
```

Comment:
For the photos, how about having 4 separate photos? like https://quarto.org/docs/authoring/figures.html#figure-panels suggests this would be easy
 Then you could replace the one with me with 1) the whiteboard photo and 2) the one photo with me where it has Two days / Time for exercises in the wordcloud (and I am looking more into camera :-D)

Reply:
I wanted to have a compact one for the thumbnail, but feel free to replace it in the article with four individual photos.

Reply:
ah yes in the article it would be good to replace it